examples/us-autocomplete-api/main.go

```diff
@@ -0,0 +1,36 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/smartystreets/smartystreets-go-sdk/us-autocomplete-api"
+	"github.com/smartystreets/smartystreets-go-sdk/wireup"
+)
+
+func main() {
+	log.SetFlags(log.Ltime)
+
+	client := wireup.NewClientBuilder().
+		WithSecretKeyCredential(os.Getenv("SMARTY_AUTH_ID"), os.Getenv("SMARTY_AUTH_TOKEN")).
```

Comment:
In the context of the go sdk, it's singular.

https://github.com/smartystreets/smartystreets-go-sdk/blob/master/secret_key_credential.go

I probably didn't think very hard about that when I authored the code and I think you're right that the plural sounds better, but it would be a breaking change so it might not be worth it.